README.md

```diff
@@ -211,18 +211,81 @@ Because the Docker container will not have direct access to your local filesyste
 To run Kombucha in Docker, first clone the repository and then do the following:
 
 ```bash
-cd kombucha
-docker build -t kombucha .
+$ cd kombucha
+$ docker build -t kombucha .
 ```
 
 This will build your Docker image and tag it with `kombucha:latest`.
 
-**NOTE:** Because the Docker container will not have direct access to your local filesystem (where you may be storing your configuration file and reference snapshots), you will need to bind a local volume to the container while it runs.
+First, let's run a quick test on your new Kombucha image to ensure everything's working properly:
 
-Let's say you have a project `sample-project` which contains the following structure:
+```bash
+$ docker run -it kombucha:latest
+```
+
+This should drop you into a bash shell on the container, with the compiled `kombucha` executable now available for direct use:
+
+```bash
+$ which kombucha
+
+/usr/local/bin/kombucha
+```
+
+Since the `Dockerfile` includes instructions to copy over the `sample.json` config file and rename it to `kombucha.json`, the program can be immediately run with no arguments, and will simply default to using `./kombucha.json` and creating snapshots in a new `__Snapshots__` directory:
+
+```bash
+$ kombucha
+
+testing: GET: httpbin.org/headers
+- 0 query param
+- 0 http header
+- empty body
+reference file doesn’t exist, writing it and failing the test…
+wrote a reference to: /app/__Snapshots__/headersSnap.json
+
+[...]
+
+finished with errors
+```
+
+**NOTE: Since this sample data contains no reference snapshots, the program will finish with errors the first time it runs, as it needs to create the snapshots first.**
+
+To avoid this first-run error, simply add the `--record` option to your arguments:
+
+```bash
+$ kombucha --record
+
+wrote a reference to: /app/__Snapshots__/headersSnap.json
+wrote a reference to: /app/__Snapshots__/ipSnap.json
+wrote a reference to: /app/__Snapshots__/responseHeadersSnap.json
+wrote a reference to: /app/__Snapshots__/userAagentSnap.json
```

Comment:
I think there’s a typo here: `Aagent`

Reply:
Oh interesting! Turns out this is actually a typo in the `sample.json` file so I will fix to reflect that 👍